library/psa_crypto.c

```diff
@@ -1390,12 +1402,25 @@ static psa_status_t psa_finish_key_creation( psa_key_slot_t *slot )
  * See the documentation of psa_start_key_creation() for the intended use
  * of this function.
  *
- * \param slot          Pointer to the slot with key material.
+ * \param[in,out] slot  Pointer to the slot with key material.
+ * \param[in] driver    The secure element driver for the key,
+ *                      or NULL for a transparent key.
  */
-static void psa_fail_key_creation( psa_key_slot_t *slot )
+static void psa_fail_key_creation( psa_key_slot_t *slot,
+                                   const psa_se_drv_table_entry_t *driver )
 {
+    (void) driver;
+
     if( slot == NULL )
         return;
+
+#if defined(MBEDTLS_PSA_CRYPTO_SE_C)
+    /* TOnogrepDO: If the key has already been created in the secure
```

Comment:
This shouldn't be here, at least not in such strange form.

Reply:
That's why it's a TODO. Spelled this way so that `check-files` doesn't tell me something I already know.

Reply:
Why not fix check-files in a separate commit instead?

Also, please add a reference in this code comment to a GitHub issue for this TODO, if it's important enough to track fixing.

Reply:
It was a conscious decision to reject `TODO` in `check-files.sh`, so I'm respecting it. Changing our criteria for acceptable code is out of scope of this PR.

Reply:
TOnogrepDO will stay in this PR. When https://github.com/ARMmbed/mbedtls/pull/2730 makes its way to this branch, we can use TODO.